spec/features/pets/index_spec.rb

```diff
@@ -81,4 +81,14 @@
     expect(page).to have_content(pet_2.name)
     expect(page).to_not have_content(pet_3.name)
   end
+
+  it "shows a link to start an application" do
+    visit "/pets"
+  
+    expect(page).to have_link("Start an Application")
+
+    click_link("Start an Application")
+
+    expect(page).to have_current_path("/applications/new")
+  end
```

Comment:
Great job adding this test here and then adding the expected behavior of the form on your applications new test.